File: [프로그래머스] 연습문제_둘만의 암호.py
def solution(s, skip, index):
    answer = ''
    for alp in s:
        loop = index
        while loop:
            alp = chr(ord(alp)+1)
            if ord(alp) > ord("z"):
                alp = "a"
            if alp in skip:
                loop += 1
            loop -= 1
        answer += alp
    return answer

File: [프로그래머스] 연습문제_둘만의 암호.js
function solution(s, skip, index) {
    let answer = "";
    let alp = 'abcdefghijklmnopqrstuvwxyz';
    for(let i of skip){
        alp = alp.replace(i, "");
    }
    let arr = alp.split("");
    for(let j of s){
        answer += arr[(arr.indexOf(j) + index) % arr.length];
    }
    return answer;
}